newIDE/app/src/ObjectEditor/Editors/SpriteEditor/AnimationPreview/index.js

```diff
@@ -1,23 +1,37 @@
 // @flow
 
 import React, { Component } from 'react';
-import { Line, Column } from '../../../../UI/Grid';
+import { Line } from '../../../../UI/Grid';
 import ImagePreview from '../../../../ResourcesList/ResourcePreview/ImagePreview';
 import Replay from 'material-ui/svg-icons/av/replay';
 import PlayArrow from 'material-ui/svg-icons/av/play-arrow';
 import Pause from 'material-ui/svg-icons/av/pause';
+import Timer from 'material-ui/svg-icons/image/timer';
+import TextField from 'material-ui/TextField';
 import { FlatButton } from 'material-ui';
 
-type Props = {
+type Props = {|
```

Comment:
@blurymind I've added flow exact notation: `{| ... |}`. This means that Flow will check against any extra state or props that you might pass/use and will error if that's the case.
I often forget to do this, but it's better to use exact notation. In particular, this help finding useless props that are passed to components (if my addition break things, will fix it in another commit)

Reply:
I see, it's more strict that way